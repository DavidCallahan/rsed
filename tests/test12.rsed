set $temp = 2
print "foo $temp"
print "foo $$temp"
print "foo \$temp"
print "foo $$"
print "$temp bar"
print "foo $temp"r
print "foo $$temp"r
print "foo \$temp"r
print "foo $$"r
