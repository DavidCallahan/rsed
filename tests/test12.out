foo 2
foo $2
foo $temp
foo $$
2 bar
foo $temp
foo $$temp
foo \$temp
foo $$
